### PR TITLE
Questions form delete

### PR DESCRIPTION
**Pull request by vrackovic**
- Added delete button, store action and API call

### CODE REVIEW

File: web/src/main/frontend/src/components/ui/DeleteButton.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from './Button';
+
+const deleteButton = props => {
+  return (
+    <div className="col-12 delete__button_container">
+      <div className="d-flex justify-content-end">
+        {props.id > 0 && (
```
Comment (iscekic) [inline web/src/main/frontend/src/components/ui/DeleteButton.js]: `props.id && ...` is enough.

Comment (iscekic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: Are we only hiding the button, or do we want to hide the whole thing (container and all)?

Comment (jolpazy) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: In current implementation of showing/hiding delete button, id is set to 0 by clear function of mobx-react-form.
So props.id && ... returns 0

Comment (vrackovic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: Yep, `props.id` is enough.
We could hide whole thing. When button is hidden, container is rendered as `div` with 100% width and no height.

Comment (iscekic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: `!!props.id`? 😄

Comment (iscekic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: I guess I'm nitpicking though, just leave this as is.

Comment (iscekic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: @vrackovic Hide the entire container and we merge. 💯

Comment (vrackovic) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: @jolpazy Aren't we using `onReset`, not `onClear`?

Comment (jolpazy) [reply web/src/main/frontend/src/components/ui/DeleteButton.js]: Whatever we are using, it leaves an id = 0

**Pull request merged**